### ISSUE TITLE
Module Setup Flow | Analytics: No existing account, existing tag

### ISSUE THREAD

**Issue by felixarntz** [google/site-kit-wp#358]
---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

Tests must cover the following:
* User clicks Activate Analytics button in Settings
* a message is shown informing about the existing tracking code and that it comes from an external (non Site Kit) source, and that the user does not have access (they need to either request access or remove the existing code)
* in this scenario, the user is **blocked** from proceeding

This is fairly similar to scenario 2 from #357

## Implementation Brief

* N/A

## Changelog entry

* N/A

**Comment by circlecube** [comment]
Little note to mention this is partially blocked by #344

**Comment by felixarntz** [comment]
@marrrmarrr Lucy pointed out the current plugin behavior in this case is different from the ACs: It shows the UI to create an account if you don't have one, regardless of whether or not there is an existing tag. Is that behavior what we want or the one outlined in the ACs?

**Comment by felixarntz** [comment]
Just to confirm, I've talked to @marrrmarrr, and she confirmed these ACs here are correct. Let's open a separate issue to make these changes.

**Comment by marrrmarrr** [comment]
> @marrrmarrr Lucy pointed out the current plugin behavior in this case is different from the ACs: It shows the UI to create an account if you don't have one, regardless of whether or not there is an existing tag. Is that behavior what we want or the one outlined in the ACs?

Confirming the ACs. In this case, even if the user creates a new account, they would still have to remove the existing tag before they're able to proceed. So having a new account won't help them.

**Comment by lucymtc** [comment]
blocked by https://github.com/google/site-kit-wp/issues/417

**Comment by aaemnnosttv** [comment]
Removing from this sprint as it likely won't be unblocked in time.

**Comment by aaemnnosttv** [comment]
Moving this back to blocked temporarily until #417 is no longer blocking.
